### ISSUE TITLE
Keepass Button on Login fields stoped working

### ISSUE THREAD

**Issue by loewexy** [keepassxreboot/keepassxc-browser#1003]
## Expected Behavior

If I go to a website, and click the keepass icon next to the username field the dialog should come up to ask me for the allowed entries.

## Current Behavior

If I click the button there is no effect.

Right click -> KeepassXC Browser -> Insert Username and Password (I tried to translate it back from German) does work.

## Steps to Reproduce (for bugs)

1. Go to some website with login form and click KeepassXC Button

## Debug info

KeePassXC - 2.6.1
KeePassXC-Browser - 1.7.0
Operating system: Ubuntu 20.04
Browser: Firefox 80.0.1

This problem started to occur at the last update. The problem is consistent on all pages I tested.

**Comment by varjolintu** [comment]
Duplicate of https://github.com/keepassxreboot/keepassxc/issues/5299. Reopen if you are certain this is not the case. If there's logins for the page, the popup should also show the icon with a question mark.

**Comment by loewexy** [comment]
I have the Best Matching option disabled. And if I go with the context menu it works without problem, so that seems to be a different bug.

**Comment by varjolintu** [comment]
Does this happen when you have one or multiple credentials for the page?

**Comment by loewexy** [comment]
It happens for both. But if I have multiple credentials for the page also the way in the context menu is broken.

**Comment by varjolintu** [comment]
Filling from the context menu works normally for me, so I'm unable to reproduce this. Try downgrading to 2.6.0 and see if it fixes it.

**Comment by loewexy** [comment]
I don't know an easy way to do this with the version from the Ubuntu PPA. I will just wait for the next release and see it that fixes ist.

**Comment by varjolintu** [comment]
Download an AppImage from the web page.

**Comment by loewexy** [comment]
That is a great Idea. I checked that and with 2.6.0 the context menu insert works in all cases. But clicking the symbol is still broken.

**Comment by loewexy** [comment]
Okay I downgraded the browser plugin to the previous version and it works again. This works also with the 2.6.1 KeepassXC

**Comment by varjolintu** [comment]
The only way to solve that is to debug the content script and see where `kpxc.fillFromUsernameIcon()` fails in `keepassxc-browser.js`.

**Comment by loewexy** [comment]
This is over my programming knowledge. So I hope someone can look into this, or it is fixed with the next release.

**Comment by varjolintu** [comment]
It's possibly already fixed, but I cannot verify that without reproducing the issue first.

**Comment by Dukat9063** [comment]
Have the same issue.
I appeared when I upgrade KeepPassXC to 2.6.1

**Comment by SilverBut** [comment]
Same problem. Alt+Shit+U still works but click the icon itself does not work. What is strange is that no Javascript is loaded from the extension's namespace. Is it expected? @varjolintu 

BTW I'm using Chromium 85.0.4183 on Linux.

![image](https://user-images.githubusercontent.com/6560655/92443839-a584c200-f1e4-11ea-8c34-4f799946f455.png)

**Comment by Ollie222** [comment]
I've just come across this issue too and it's affecting Firefox and Chrome using KeepassXC 2.6.0 and 2.6.1.

Clicking the icon in the username doesn't appear to be requesting the credentials from the database but if you have the setting 'Automatically retrieve credentials.' ticked then clicking the icon does list them.

**Comment by varjolintu** [comment]
@Ollie222 That bug has been already fixed.

**Comment by varjolintu** [comment]
@SilverBut You need to click the double arrow icond and select Content Scripts.

**Comment by Ollie222** [comment]
> 
> 
> @Ollie222 That bug has been already fixed.

Thanks for the reply.

Does the release version 1.7.0 contain this fix as I have that and KeepassXC 2.6.1 and I still have the problem.

**Comment by varjolintu** [comment]
@Ollie222 It's in 1.7.1. You can try https://github.com/keepassxreboot/keepassxc-browser/pull/1000 for testing it manually.

**Comment by Ollie222** [comment]
@varjolintu Thanks for letting me know and good to know the issue is in hand. I'll look out for the new version when it drops.

**Comment by SilverBut** [comment]
> @SilverBut You need to click the double arrow icond and select Content Scripts.

Thanks for help. I succeed to find out that in `kpxc.fillFromUsernameIcon`, `kpxc.credentials.length === 0` and returned directly. It's kind of strange since I am not sure who wrote this variable....let me try again. Not really good at javascript.

**Comment by SilverBut** [comment]
@varjolintu Maybe I found partial reason of this issue...

When ask for filling credentials by using keyboard or context menu (in `manifest.json:80`), message `fill_username_password` will be issued. However, for some reason, when clicking the icon in username field, no new messages is issued to the anonymous function defined in `browser.runtime.onMessage.addListener`. Handler for that message calls `await kpxc.receiveCredentialsIfNecessary();` so, if this method is not called, the extension is not even asking background page (and KeePassXCProxy) for matchiung URLs. 

I'm not quite sure about how those messages are delivered to those listeners, and also not sure about who generated those messages...maybe something wrong for the button's `onClick()`? Not implemented?

**Comment by varjolintu** [comment]
@SilverBut See https://github.com/keepassxreboot/keepassxc-browser/pull/990 :)

**Comment by SilverBut** [comment]
> @SilverBut See #990 :)

Oh saw this seconds after I wrote that reply....seems fixed. Waiting for next release, thanks!

**Comment by baekelite** [comment]
I pulled down the 1.7.1 fixes and I can confirm these problems are fixed 👍 

platform: macOS browser: firefox
